x/ethbridge/types/querier.go

```diff
@@ -10,25 +10,22 @@ const (
 )
 
 // NewQueryEthProphecyRequest creates a new QueryEthProphecyParams
-func NewQueryEthProphecyRequest(
-	ethereumChainID int64, bridgeContractAddress EthereumAddress, nonce int64, symbol string,
-	tokenContractAddress EthereumAddress, ethereumSender EthereumAddress,
-) *QueryEthProphecyRequest {
+func NewQueryEthProphecyRequest(prophecyID []byte) *QueryEthProphecyRequest {
 	return &QueryEthProphecyRequest{
-		EthereumChainId:       ethereumChainID,
-		BridgeContractAddress: bridgeContractAddress.String(),
-		Nonce:                 nonce,
-		Symbol:                symbol,
-		TokenContractAddress:  tokenContractAddress.String(),
-		EthereumSender:        ethereumSender.String(),
+		ProphecyId: prophecyID,
 	}
 }
 
 // NewQueryEthProphecyResponse creates a new QueryEthProphecyResponse instance
-func NewQueryEthProphecyResponse(id string, status oracletypes.Status, claims []*EthBridgeClaim) QueryEthProphecyResponse {
+func NewQueryEthProphecyResponse(id []byte, status oracletypes.StatusText, claims []string) QueryEthProphecyResponse {
+	// claimValidators := []string{}
```

Comment:
Remove commented out code assuming we don't need it.

Reply:
accepted.